cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -425,6 +426,21 @@ public ClusterOperationMetadata createCluster(String name, String imageVersion,
             .build());
       }
 
+      //Set Auto Scaling Policy
+      if (conf.isAutoPolicyEnabled()) {
+        String policyURI = conf.getAutoScalingPolicy();
+        //Check if policy is URI or ID. If ID Convert to URI
+        if (!policyURI.contains("/")) {
```

Comment:
Can this me more strict, is it looking for `/` at start or in-between. My guess is it's looking for `^/`

Reply:
Its just looking for a single / as the URI does not have to start with it.
From Dataproc docs - 
 
--autoscaling-policy flag to specify the policy id (the name of the policy you specified when you created the policy) or the policy resource URI (resource name). 

The URI can be a full Resource name
"//library.googleapis.com/shelves/shelf1/books/book2"
or 
Relative resource name 
"shelves/shelf1/books/book2"